src/index.html

```diff
@@ -4,13 +4,54 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="searchbar_form"
+      data-qa="big"
+    >
+      <div class="big_searchbar searchbar">
+        <img
+          src="images/Search.svg"
+          alt="Search icon"
+          class="big_icon search_icon"
```

Comment:
as for me - more clear to set generic class first and then specific for this size